packages/@o3r/styling/ng-package.json

```diff
@@ -12,6 +12,7 @@
     "scss/**/*",
     "!scss/**/*.spec.scss",
     "assets/**",
-    "_index.scss"
+    "_index.scss",
+    "_otter-theme.scss"
```

Comment:
[no change required comment]
Just hoping we won't have hard time to debug the files exposed by node resolution exports mapping and the deep import to the scss file :S